app/mailers/notifications.rb

```diff
@@ -88,15 +88,26 @@ def consultation_deadline_passed(consultation)
       subject: "Consultation deadline breached"
   end
 
+  helper_method :production?
+
+  def production?
+    /production/.match?(GovukAdminTemplate.environment_style)
```

Comment:
Can this just be
```suggestion
    GovukAdminTemplate.environment_style == "production"
```
now?

Reply:
Yeah, I thought about that. I wasn’t sure if there was a good reason for it being a regex other than the case insensitive comparison with a human readable string so decided to leave it, but perhaps it’s worth giving it a go.

Reply:
I've amended the PR with this change now and all the tests still seem to be passing!